crates/shielder-cli/src/app_state.rs

```diff
@@ -36,6 +36,10 @@ impl RelayerRpcUrl {
         format!("{}/quote_fees", self.base_url)
```

Comment:
Is this change supposed to be a part of the browser extension release? If yes, after merging we should publish the mpm using [instructions](https://www.notion.so/cardinalcryptography/Publishing-SDK-Relayer-and-Dev-Environment-134af09db1a980ab9bf2e04f8591a78a) and [workflow](https://github.com/Cardinal-Cryptography/zkOS-monorepo/actions/workflows/manual-publish-dev-env.yml), and then coordinate with Piotr Dżaluk to make sure the change is included in browser extension code.